pkg/oam/util/test_utils.go

```diff
@@ -132,6 +132,19 @@ func (matcher ErrorMatcher) NegatedFailureMessage(actual interface{}) (message s
 	return format.Message(actual, "not to equal", expectedError)
 }
 
+// UnMarshalStringToComponentDefinition parse a string to a componentDefinition object
+func UnMarshalStringToComponentDefinition(s string) (*v1alpha2.ComponentDefinition, error) {
```

Comment:
why not just replace the func `UnMarshalStringToWorkloadDefinition`, does it still be used?

Reply:
UnMarshalStringToWorkloadDefinitio will not be used, i will remove it